packages/salesforcedx-vscode-lwc/src/messages/i18n.ts

```diff
@@ -84,6 +84,11 @@ export const messages = {
   force_lightning_lwc_ios_failure: "Failed to start iOS Simulator '%s'.",
   force_lightning_lwc_android_start: "Starting Android Emulator '%s'.",
   force_lightning_lwc_ios_start: "Starting iOS Simulator '%s'.",
+  force_lightning_lwc_preview_create_virtual_device_label: 'New...',
```

Comment:
We got doc sign-off on these strings, right?

Reply:
yes got them from Richard.